_specifications/specification-3-16.md

```diff
@@ -249,6 +249,8 @@ scheme     authority       path        query   fragment
 
 We also maintain a node module to parse a string into `scheme`, `authority`, `path`, `query`, and `fragment` URI components. The GitHub repository is [https://github.com/Microsoft/vscode-uri](https://github.com/Microsoft/vscode-uri) the npm module is [https://www.npmjs.com/package/vscode-uri](https://www.npmjs.com/package/vscode-uri).
 
+On case-insensitive file systems, two textually different URIs might represent the same file.

```

Comment:
Could this be rephrased to avoid leading someone to conclude that on case-sensitive file systems, textually different URIs represent different files?

These URIs are textually different but [*equivalent*](https://url.spec.whatwg.org/#url-equivalence) despite using only one case:
- `file:///foo/bar`
- `file:///foo/./bar`
- `file:///foo/%62ar`
- `file://localhost/foo/bar

Even ignoring equivalence, is this not also true of all file systems that allow symlinks or multiple hard links?

Reply:
I guess at this point it stalled long enough to just close.